### PR TITLE
update build action closes #14

### LINKED ISSUE #14: Setup CI environment
First try is [Github CI](https://docs.github.com/en/free-pro-team@latest/actions/guides/about-continuous-integration) 
If doesn't cover our needs, another try is Travis, Circle CI
Add status badge in a main project readme file.

### PR DESCRIPTION
**Pull request by saulyz**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  test: # make sure the action works on a clean machine without building
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```
Comment (AzisK) [inline .github/workflows/test.yml:20]: As I am not very familiar with Github CI system and deno, what does this line actually do?

**Review by benNek — APPROVED** [review]
Looks good

File: .github/workflows/test.yml (hunk 1 above)
Comment (psaulenas) [reply .github/workflows/test.yml:20]: https://github.com/actions/checkout#checkout-v2

Comment (AzisK) [reply .github/workflows/test.yml:20]: Thanks!

**Review by AzisK — APPROVED** [review]
Looking good

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# This workflow uses actions that are not certified by GitHub.
```
Comment (psaulenas) [inline .github/workflows/test.yml:1]: Do we need these both workflows? Maybe just `build-test` would be enough?

**Review by psaulenas — APPROVED** [review]
Overall looks good to me, just a concern if we really need two workflows because there is a limit for Github Actions minutes.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](CODE_OF_CONDUCT.md)
 
+![test](https://github.com/pets-oss/pets-front/workflows/test/badge.svg?branch=main)
```
Comment (psaulenas) [inline README.md:3]: We should change this also:

```suggestion
![test](https://github.com/pets-oss/pets-front/workflows/integrate/badge.svg?branch=main)
```

**Pull request merged**